js/plugins.js

```diff
@@ -6,115 +6,139 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import {toModulePlugin} from "@mapstore/utils/ModulePluginsUtils";
+
+import Context from "@mapstore/plugins/Context";
+import ContextCreator from "@mapstore/plugins/ContextCreator";
+import Dashboard from "@mapstore/plugins/Dashboard";
+import Dashboards from "@mapstore/plugins/Dashboards";
+import FeedbackMask from '@mapstore/plugins/FeedbackMask';
+import GeoStory from "@mapstore/plugins/GeoStory";
+import GeoStories from "@mapstore/plugins/GeoStories";
+import Maps from "@mapstore/plugins/Maps";
+import RulesDataGrid from "@mapstore/plugins/RulesDataGrid";
+import RulesEditor from "@mapstore/plugins/RulesEditor";
+import RulesManagerFooter from "@mapstore/plugins/RulesManagerFooter";
+import Print from "@mapstore/plugins/Print";
+
 /**
   * Please, keep them sorted alphabetically
  */
-module.exports = {
+export default {
```

Comment:
I thought about similar approach, the only thing that stopped me from reformatting is that I've never seen it used anywhere 👍 
I've applied slightly another version, adding filtering because not all plugins from upstream were listed in geOrchestra's plugins.js file.